### DISCUSSION TITLE
additional hepatic clearance

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1238] [Questions & Problems]
How to add additional hepatic clearance in PK-sim？I can't find the corresponding position.
![额外肝清除](https://user-images.githubusercontent.com/95362728/176065284-5451bc68-16a8-4986-a158-3bbe6c2fff9a.jpg)
![软件截图](https://user-images.githubusercontent.com/95362728/176065298-41c83198-249b-4a95-9b24-ee491dc72ca9.jpg)

**Comment by StephanSchaller** [comment]
this is referring to a non-specific "total hepatic clearance" process (the last entry in your screensgot)

**Reply by WTY-tt** [reply]
Thank you for your answer. My question is whether I should fill in the total hepatic clearance rate or the additional hepatic clearance rate except CYP enzyme

**Reply by StephanSchaller** [reply]
If I undestand it correct, you should fil in both. the additional hepatic CL is for covering nonspecified enzymatic CL processes.

**Reply by fredsevmartins** [reply]
@WTY-tt  You have 2 metabolism CL. 1) mediated by enzymes and you should enter those values at " metabolizing enzyme".
2) The additional hepatic CL should be entered at  " Total hepatic Clearance" , remember to convert the unit l/hr to l/h/kg.

**Reply by WTY-tt** [reply]
Thank you very much.
